src/datasetManifest.js

```diff
@@ -251,7 +249,85 @@ export const allDatasets = [
     path: "datasets/medical_priority.csv",
     metadataPath: "datasets/medical_priority.json",
     imagePath: "datasets/temp.jpg"
+  },
+  {
+    id: "nutrition",
```

Comment:
@brondle beep boop - looks like nutrition shows up twice? Here and line 261?

Reply:
whoops should be fixed